spec/classes/apt_transport_https.rb

```diff
@@ -0,0 +1,15 @@
+require 'spec_helper'
+describe 'apt::transport_https', :type => :class do
+  let(:facts) { { :lsbdistid => 'Debian', :osfamily => 'Debian', :lsbdistcodename => 'wheezy', :puppetversion   => Puppet.version, } }
```

Comment:
I'd prefer to see this hash on multiple lines and aligned, makes reading easier.